cmd/osbuild-composer/composer.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/osbuild/osbuild-composer/internal/distro/fedora33"
 	"github.com/osbuild/osbuild-composer/internal/distro/rhel8"
 	"github.com/osbuild/osbuild-composer/internal/distro/rhel84"
+	"github.com/osbuild/osbuild-composer/internal/distro/rhel9"
```

Comment:
let's call this `rhel90`, to avoid repeating the mistake of `rhel8` (which really is `rhel83`)

Reply:
Or maybe go even deeper and call it `rhel90beta`?